hugr-model/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+//! The data model of the HUGR intermediate representation.
+//! This crate defines data structures that capture the structure of a HUGR graph and
+//! all its associated information in a form that can be stored on disk. The data structures
```

Comment:
nit: does anyone still use "disk"s nowadays? :)

perhaps something like "in a form designed for simple and efficient serialization, rather than traversal or modification"??? Or we could say this is for serialization, storage *and interchange*?

Reply:
I think it's used metaphorically, like the floppy disk symbol meaning "save" although it's most likely not being saved on a floppy disk.